react/features/invite/components/add-people-dialog/web/InviteContactsForm.js

```diff
@@ -0,0 +1,489 @@
+// @flow
+import InlineMessage from '@atlaskit/inline-message';
+import React from 'react';
+import type { Dispatch } from 'redux';
+
+import AbstractAddPeopleDialog, {
+    type Props as AbstractProps,
+    type State,
+    _mapStateToProps as _abstractMapStateToProps
+} from '../AbstractAddPeopleDialog';
+import { Avatar } from '../../../../base/avatar';
+import { connect } from '../../../../base/redux';
+import { getLocalParticipant } from '../../../../base/participants';
+import { Icon, IconPhone } from '../../../../base/icons';
+import { MultiSelectAutocomplete } from '../../../../base/react';
+import { translate, translateToHTML } from '../../../../base/i18n';
+
+declare var interfaceConfig: Object;
+
+type Props = AbstractProps & {
+
+    /**
+     * The {@link JitsiMeetConference} which will be used to invite "room"
+     * participants through the SIP Jibri (Video SIP gateway).
```

Comment:
It's actually Jigasi. No need to mention how we invite them, since this may change in the future.